src/averageAges.js

```diff
@@ -14,12 +14,18 @@
  *
  * @return {number}
  */
+const middleAge = (people) => {
+  return people.reduce((count, person) =>
+    count + (person.died - person.born), 0) / people.length;
+};
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => century
```

Comment:
Let's use more reliable check here, because century may theoretically have valid falsy value (0)
```suggestion
  const men = people.filter(person => century !== undefined
```